ftplugin/rest.vim

```diff
@@ -826,6 +826,7 @@ function! VrcGetRequestPath()
       echom request.msg
   endif
 endfunction
+nnoremap <buffer> <silent> gx :silent! call system("open " .VrcGetRequestPath())<cr>
```

Comment:
Not all systems have the `open` command. You can still remap this in `.vimrc` though.